ETC.tsv

```diff
@@ -2759,7 +2759,7 @@ ETC_20150317_002758	Possible
 ETC_20150317_002759	Not Available
 ETC_20150317_002760	Critical Attack
 ETC_20150317_002761	Critical Defense
-ETC_20150317_002762	Tradeable
+xETC_20150317_002762	Tradable
```

Comment:
They were a light edit but i will fix those you mention. Yes , that one is a spelling issue. Is something wrong with it?